package.json

```diff
@@ -9,9 +9,11 @@
     "@mui/material": "^5.10.15",
     "@mui/styled-engine-sc": "^5.10.14",
     "@openapitools/openapi-generator-cli": "^2.5.2",
+    "@react-google-maps/api": "^2.17.0",
     "@testing-library/jest-dom": "^5.16.5",
     "@testing-library/react": "^13.4.0",
     "@testing-library/user-event": "^13.5.0",
+    "axios": "^1.2.0",
```

Comment:
where is axios used?

Reply:
I thought I needed it at one point, then forgot to uninstall it. removed